nix/haskell.nix

```diff
@@ -109,21 +109,9 @@ let
           unit.build-tools = [ jmPkgs.jormungandr ];
         };
 
-        # Add jormungandr to the PATH of the latency benchmark
-        packages.cardano-wallet-jormungandr.components.benchmarks.latency = wrapBench jmPkgs.jormungandr;
-        packages.cardano-wallet.components.benchmarks.latency =
-          lib.optionalAttrs (!stdenv.hostPlatform.isWindows) {
-            build-tools = [ pkgs.makeWrapper ];
-            postInstall = ''
-              wrapProgram $out/bin/* \
-                --run "cd $src" \
-                --prefix PATH : ${pkgs.cardano-node}/bin
-
-              wrapProgram $out/bin/* \
-                --run "cd $src" \
-                --prefix PATH : ${pkgs.cardano-cli}/bin
-            '';
-          };
+        # Add node backend to the PATH of the latency benchmarks
+        packages.cardano-wallet-jormungandr.components.benchmarks.latency = wrapBench [ jmPkgs.jormungandr ];
+        packages.cardano-wallet.components.benchmarks.latency = wrapBench [ pkgs.cardano-node pkgs.cardano-cli ];
```

Comment:
That was my original intention but didn't know how to put that in `wrapBench` :(

Reply:
No worries